proteuslib/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -313,6 +325,10 @@ def test_initialize(self, RO_frame):
     def test_var_scaling(self, RO_frame):
         m = RO_frame
         badly_scaled_var_lst = list(badly_scaled_var_generator(m))
+        for (var, v) in badly_scaled_var_lst:
```

Comment:
It looks like this was here for debugging purposes and should be removed (although I am not against leaving it in).